docs/content/index.md

```diff
@@ -2,60 +2,153 @@
 layout: doc_page
 ---
 
-# About Druid
+Introduction to Druid
+---------------------
 
-Druid is an open-source analytics data store designed for OLAP queries on timeseries data (trillions of events, petabytes of data). Druid provides cost-effective and always-on real-time data ingestion, arbitrary data exploration, and fast data aggregation.
+Druid is an open source data store designed for [OLAP](http://en.wikipedia.org/wiki/Online_analytical_processing) queries on timeseries data.
```

Comment:
Why'd you delete the pitch about what we think the best attributes of Druid are? I think that should be first.